composer.json

```diff
@@ -52,7 +52,8 @@
         "symfony/yaml": "^2.3 || ^3.0"
     },
     "conflict": {
-        "jms/di-extra-bundle": "<1.7.0"
+        "jms/di-extra-bundle": "<1.7.0",
+        "sonata-project/exporter": "1.6.0"
```

Comment:
It's just `1.6.0` that conflicts: it does not have the `getAvailableFormats()` method and I don't want to also check for that. for `1.5` for instance, the code will behave just like it did before.